### PR TITLE
Flutter 2.2.0 Instructions

### PR DESCRIPTION
**Pull request by offlineprogrammer**
*Issue #, if available:* [Instructions for Flutter 2.2.0 compatibility with Amplify](https://github.com/aws-amplify/docs/issues/3258)

*Description of changes:* Add instructions to disable sound null safety

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/analytics/fragments/flutter/getting-started/10_preReq.md [hunk 1]
```diff
@@ -3,3 +3,9 @@
     * An iOS configuration targeting at least iOS 11.0
     * An Android configuration targeting at least Android API level 21 (Android 5.0) or above
     * For a full example of please follow the [project setup walkthrough](~/lib/project-setup/create-application.md)
+
+<amplify-callout warning>
+
+If you are using Flutter (2.2.0), you need disable disable sound null safety for by following the instructions [here](https://dart.dev/null-safety/unsound-null-safety#testing-or-running-mixed-version-programs). Null safety support for Amplify Flutter is being worked on actively.
```
Comment (Jordan-Nelson) [inline docs/lib/analytics/fragments/flutter/getting-started/10_preReq.md:9]:
```suggestion
If you are using Flutter (2.2.0), you need disable disable sound null safety by following the instructions [here](https://dart.dev/null-safety/unsound-null-safety#testing-or-running-mixed-version-programs). Null safety support for Amplify Flutter is being worked on actively.
```

Comment (Jordan-Nelson) [reply docs/lib/analytics/fragments/flutter/getting-started/10_preReq.md:9]: Same suggestion applies to the other callouts

**Review by Jordan-Nelson — APPROVED** [review]

File: docs/lib/analytics/fragments/flutter/getting-started/10_preReq.md [hunk 2]
```diff
@@ -3,3 +3,9 @@
     * An iOS configuration targeting at least iOS 11.0
     * An Android configuration targeting at least Android API level 21 (Android 5.0) or above
     * For a full example of please follow the [project setup walkthrough](~/lib/project-setup/create-application.md)
+
+<amplify-callout warning>
+
+If you are using Flutter (2.2.0), you need to disable sound null safety by following the instructions [here](https://dart.dev/null-safety/unsound-null-safety#testing-or-running-mixed-version-programs). Null safety support for Amplify Flutter is being worked on actively.
```
Comment (ragingsquirrel3) [inline docs/lib/analytics/fragments/flutter/getting-started/10_preReq.md:9]: non-blocking nit: Does it make sense to change this to "(2.2.0 or greater)" to clarify that it's not this specific version of flutter, but all the versions after (presumably)? Maybe this is obvious to most readers but a small clarification.

**Review by ragingsquirrel3 — COMMENTED** [review]
LGTM w 1 nit/question.

File: docs/lib/analytics/fragments/flutter/getting-started/10_preReq.md (hunk 2 above)
Comment (ragingsquirrel3) [reply docs/lib/analytics/fragments/flutter/getting-started/10_preReq.md:9]: nm, I see this merged

**Pull request merged**